android/modules/ui/src/java/ti/modules/titanium/ui/TabProxy.java

```diff
@@ -25,6 +25,7 @@
 		TiC.PROPERTY_ACTIVE_TITLE_COLOR,
 		TiC.PROPERTY_BADGE,
 		TiC.PROPERTY_BADGE_COLOR,
```

Comment:
Can we deprecate this one and move it to `badgeBackgroundColor` for improved consistency to other similar APIs?

Reply:
sure. update the PR. It will show a warning to use `badgeBackgroundColor` but both values will continue to work